.github/workflows/deployment.yml

```diff
@@ -11,24 +13,38 @@ on:
 
 jobs:
 
-  build:
+  Version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.daily-version.outputs.version }}
+      created: ${{ steps.daily-version.outputs.created }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: fregante/daily-version-action@v1
+        name: Create tag if necessary
+        id: daily-version
```

Comment:
1. Outputs must be declared
1. `fregante/daily-version-action` was tested on:
	- push
		- commit
			> HEAD isn’t tagged. `daily-version-action` will create `20.6.6.1327`
		- tag
			> Run triggered by tag `20.6.6.1321`. No new tags will be created by `daily-version-action`.
	- any non-push event:
		- needs tag
			> HEAD isn’t tagged. `daily-version-action` will create `20.6.6.1326`
		- doesn't need tag
			> No new commits since the last tag (20.6.6.1324). No new tags will be created by `daily-version-action`.